src/fs/file.rs

```diff
@@ -354,6 +379,11 @@ impl<'dir> File<'dir> {
         }
     }
 
+    #[cfg(windows)]
+    pub fn changed_time(&self) -> Option<SystemTime> {
+        return self.modified_time()
```

Comment:
I don't quite understand the difference between `changed_time` & `modified_time`. Nor do I know if `changed_time` exists on Windows. So I just did this.

Reply:
Ah, it's the time the file attributes change rather than the file's data. I don't think it does exist on Windows.